blueprint/index.md

```diff
@@ -7,232 +7,285 @@ icon: blueprint
+5. Click **Properties**.
+6. Enter the endpoint URL that you saved from the Amazon API Gateway in the [Bot Connector Handle Utterance URI](#create-a-power-va-bot "Opens the Create a Power VA bot section") property.
+For example, `https://accountname.execute.api.eu.west-1.amazonaws.com/default/postutterance`
 	
 	![Post Utterance Setting](images/PostUtteranceSettings.png "Post Utterance Setting")
-	5. Under the Credentials tab, we must add something. You can use the bearer we created earlier, or any authorization headers you want to send.
-	6. Click Configure. Click Add Credential Field. You can enter Authorization for the Field Name and “Bearer {YOUR SECRET 1}” for the Value. The sample does not use this, but for other integrations it can help
-
-	![Configure Credentials](images/Credentials.png "Configure Credentials")
-	7. Save and activate the integration
-	8. Get the ID for the integration from the URL as we will need this later.
-2. Create a role (or edit an existing one) and add the permissions to it:
-	1. Integration -> botconnector -> All Permissions
-	2. Textbots -> All Permissions
-	
-	![GC Permissions](images/GCPermissions.png "GC Permissions")
-	3. Assign this role to your account in Genesys Cloud.
-	
-###Declare the Power VA bot to Genesys Cloud
-
-1. Get the PVA bot id from the Microsoft PowerVA (after “Default”) URL: https://web.powerva.microsoft.com/environments/Default-xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/bots/{id}/ as mentioned earlier.
-2. Get your access token from Genesys Cloud to authenticate against the API.
-3. Call the following Genesys Cloud API (check if topics and entities are correct first):
-
-PUT https://api.mypurecloud.ie/api/v2/integrations/botconnector/{YOUR GENESYS BOT CONNECTOR INTEGRATION ID}/bots
-Authorization: bearer {YOUR GC ACCESS TOKEN}
-Content-Type: application/json
-
-{
-  "chatBots": [
-    {
-      "id": "{PVA BOT ID}",
-      "name": "{DISPLAY NAME}",
-      "versions": [
-        {
-          "version": "Delta",
-          "supportedLanguages": [
-            "en-us",
-            "es"
-          ],
-          "intents": [
-            {
-              "name": "Check Stock",
-              "slots": {
-                "Slot1 ": {
-                  "name": "Slot1",
-                  "type": "String"
-                }
-              }
-            },
-            {
-              "name": "Escalate",
-              "slots": {
-                "Slot2": {
-                  "name": "Slot2",
-                  "type": "String"
-                }
-              }
-            }
-          ]
-        }
-      ]
-    }
-  ]
-}
-
-4. You should receive a 204 No Content if all works well
-5. This declares your bot config to Genesys Cloud so it can be found in Architect.
-
-###Create an Architect Flow
-
-1. In Architect, create a new Inbound Message flow (no other type is supported).
-2. From the Tools, drag on a Call Bot Connector block from the Bot menu
```

Comment:
I can't see the heading for the following section Should this be ###Create an Architect flow?